data/json/items/armor/legs_armor.json

```diff
@@ -79,10 +79,34 @@
     "encumbrance": 24,
     "max_encumbrance": 30,
     "pocket_data": [
-      { "pocket_type": "CONTAINER", "max_contains_volume": "500 ml", "max_contains_weight": "1 kg", "moves": 80 },
-      { "pocket_type": "CONTAINER", "max_contains_volume": "500 ml", "max_contains_weight": "1 kg", "moves": 80 },
-      { "pocket_type": "CONTAINER", "max_contains_volume": "250 ml", "max_contains_weight": "1 kg", "moves": 100 },
-      { "pocket_type": "CONTAINER", "max_contains_volume": "250 ml", "max_contains_weight": "1 kg", "moves": 100 }
+      {
+        "pocket_type": "CONTAINER",
+        "max_contains_volume": "3250 ml",
```

Comment:
Big changes here.

Reply:
The turnout pants I based this off of have the same 10x10 pockets as the coat does.  Two back pockets as well.